packages/NodeTypeResolver/tests/PerNodeTypeResolver/ClassTypeResolver/ClassTypeResolverTest.php

```diff
@@ -42,10 +42,11 @@ public function dataProvider(): Iterator
 
         yield [__DIR__ . '/Source/ClassWithTrait.php', 0, [ClassWithTrait::class, AnotherTrait::class]];
 
+        // traits in anonymous classes are ignored in PHPStan
         yield [
             __DIR__ . '/Source/AnonymousClass.php',
             0,
-            [ParentClass::class, SomeInterface::class, AnotherTrait::class],
+            [ParentClass::class, SomeInterface::class /*, AnotherTrait::class */],
```

Comment:
Maybe remove this test?

Reply:
Why?

Reply:
It’s commented.

Reply:
Oh, you mean just the part, not the whole test.

Since it's edge case PHPStan skips, I think we can remove that part. Will do